src/index.ts

```diff
@@ -0,0 +1,259 @@
+/**
+ * Build styles
+ */
+import './index.css';
+
+import { IconText } from '@codexteam/icons'
+import { API, BlockTool, PasteEvent } from '@editorjs/editorjs';
+import type { ParagraphToolConfig, ParagraphToolCSS, ParagraphToolData, PasteConfig, Toolbox } from './types';
+
+/**
+ * Base Paragraph Block for the Editor.js.
+ * Represents simple paragraph
+ *
+ * @author CodeX <team@codex.so>
+ * @copyright CodeX 2023
+ * @license The MIT License (MIT)
+ */
+export default class Paragraph implements BlockTool {
+  /**
+   * Code API — public methods to work with Editor
+   *
+   * @link https://editorjs.io/api
+   */
+  private readonly api: API;
+
+  /**
+   * Read-only mode flag
+   */
+  private readonly readOnly: boolean;
+
+  /**
+   * Tool data for input and output
+   */
+  private _data: ParagraphToolData = {};
```

Comment:
```suggestion
  private data: ParagraphToolData = {};
```

Reply:
There are already getters and setters for `data`. So, there is no point in changing.